packages/lib/src/components/internal/IbanInput/IbanInput.tsx

```diff
@@ -104,14 +109,16 @@ class IbanInput extends Component<IbanInputProps, IbanInputState> {
         const iban = formatIban(electronicFormatIban); // example: NL13 TEST 0123 4567 89
         const validationStatus = checkIbanStatus(iban).status;
 
+        const countryCode = getCountryCode(electronicFormatIban);
+
         // calculate cursor's new position
         const cursor = e.target.selectionStart;
-        const previousIban = this.state.data['sepa.ibanNumber'];
+        const previousIban = this.state.data['ibanNumber'];
         const newCursorPosition = getNextCursorPosition(cursor, iban, previousIban);
 
         this.setState(
             prevState => ({
-                data: { ...prevState.data, 'sepa.ibanNumber': iban },
+                data: { ...prevState.data, 'ibanNumber': iban, 'countryCode': countryCode },
```

Comment:
Minor thing: no need for single quotes here. And you can just pass the variable `countryCode` , you don't need to define the name and assign to it if they have the same name